apps/condo/bin/delete-meter-readings-by-property-ids.js

```diff
@@ -0,0 +1,87 @@
+const path = require('path')
+const { find } = require('@core/keystone/schema')
+const { GraphQLApp } = require('@keystonejs/app-graphql')
+const get = require('lodash/get')
+const { MeterReading, Meter } = require('@condo/domains/meter/utils/serverSchema')
+
+class FixMeterReadingsClients {
+    constructor (propertyIds) {
+        this.propertyIds = propertyIds
+        this.context = null
+        this.meters = []
+        this.meterReadings = []
+    }
+
+    async connect () {
+        const resolved = path.resolve('./index.js')
+        const { distDir, keystone, apps } = require(resolved)
+        const graphqlIndex = apps.findIndex(app => app instanceof GraphQLApp)
+        await keystone.prepare({ apps: [apps[graphqlIndex]], distDir, dev: true })
+        await keystone.connect()
+        this.context = await keystone.createContext({ skipAccessControl: true })
+    }
+
+    async findMeterReadings () {
+        this.meterReadings = await find('MeterReading', {
+            meter: { property: { id_in: this.propertyIds } },
+        })
+    }
+
+    async findMeters () {
+        this.meters = await find('Meter', {
+            property: { id_in: this.propertyIds },
+        })
+    }
+
+    async deleteBrokenMeterReadings () {
+        if (!get(this.meterReadings, 'length')) return
+
+        for (const meterReading of this.meterReadings) {
+            await MeterReading.update(this.context, meterReading.id, {
```

Comment:
We have `softDelete` method in `generate.server.utils.js` module.

Reply:
added softDelete method call